qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/kubernetes.tf

```diff
@@ -158,6 +158,18 @@ module "kubernetes-ingress" {
 
   node-group = local.node_groups.general
 
+{% if cookiecutter.load_balancer_ip is defined %}
+  load-balancer-ip = "{{ cookiecutter.load_balancer_ip }}"
+{% endif %}
+
+{% if cookiecutter.load_balancer_annotations is defined %}
+  load-balancer-annotations = {
+{% for key, value in cookiecutter.load_balancer_annotations.items() %}
+    {{ "%-10s" | format(key) }} = "{{ value }}"
```

Comment:
Should be:
`"{{ "%-10s" | format(key) }}" = "{{ value }}"`

Reply:
Oh that's right! Yeah I didn't consider the keys that kubernetes uses in annotations. Thanks for the catch!

Reply:
Fixed.